test/test-wikipedia-kyoto-japanese-english.rb

```diff
@@ -0,0 +1,95 @@
+class WikipediaKyotoJapaneseEnglishTest < Test::Unit::TestCase
+  def setup
+    @dataset_articles = Datasets::WikipediaKyotoJapaneseEnglish.new(type: :article)
+    @dataset_lexicon = Datasets::WikipediaKyotoJapaneseEnglish.new(type: :lexicon)
+  end
+
+  test('#article') do
```

Comment:
Ruby uses `#XXX` as a notation for `XXX` method. So this usage isn't popular. How about using just `article`?

Reply:
I reflected this change and will remember this. Thank you for pointing this out.